### ISSUE TITLE
Clustered bar chart?

### ISSUE THREAD

**Issue by smashmirrorcardboardface** [deneb-viz/deneb#112]
is there any way to get 2 data fields to cluster like the standard PBI clustered bar chart using deneb / vega-lite?

**Comment by dm-p** [comment]
I'm going to be closing this as a simple way of managing (with x/y offset) is on the VL roadmap and is currently a language limitation. The above example has [now been published to the main Vega-Lite examples site](https://vega.github.io/vega-lite/examples/bar_grouped.html), which may assist with this particular challenge.

I've done a quick extension of this example in the visual and while I haven't gone completely all the way there, this looks okay (with the unfortunate exception of having to prescribe step width and height due to faceting):

![image](https://user-images.githubusercontent.com/10572054/132792941-18aa15ed-39d7-42f3-94b4-6c15ffddffac.png)

Spec for this as follows (with `config` in-lined):

```json
{
  "data": {"name": "dataset"},
  "width": {"step": 15},
  "height": 250,
  "config": {
    "view": {"stroke": "transparent"},
    "font": "Segoe UI",
    "header": {"title": null},
    "axis": {
      "grid": false,
      "domain": false,
      "ticks": false,
      "tickCount": 5,
      "labelPadding": 10,
      "title": null
    },
    "axisX": {"disable": true},
    "legend": {
      "orient": "top",
      "title": null
    }
  },
  "mark": {
    "type": "bar",
    "tooltip": true
  },
  "encoding": {
    "column": {
      "field": "Country",
      "header": {"orient": "bottom"}
    },
    "color": {"field": "Product"},
    "x": {"field": "Product"},
    "y": {
      "field": "$ Sales",
      "type": "quantitative",
      "axis": {
        "format": "$#0,,,.0bn",
        "formatType": "pbiFormat"
      }
    }
  }
}
```

**Comment by smashmirrorcardboardface** [comment]
nice work.  thank you.

**Comment by dm-p** [comment]
Although this is closed, Vega-Lite 5.2.0 has the new offset channel is added in 5.2.0, including 2 examples:

- [Grouped Bar Chart](https://vega.github.io/vega-lite/examples/bar_grouped.html)
- [Grouped Bar Chart(with repeat)](https://vega.github.io/vega-lite/examples/bar_grouped_repeated.html)

5.2.0 ia scheduled for #158 / v1.1
